h/api/models/document.py

```diff
@@ -40,7 +44,7 @@ def title(self):
     @classmethod
     def find_by_uris(cls, session, uris):
         """Find documents by a list of uris."""
-        query_uris = [text_type(uri.normalize(u), 'utf-8') for u in uris]
+        query_uris = [normalize(u) for u in uris]
```

Comment:
I'll have a go at it. It should be in its own PR anyway, since almost all of the 4 PG pull requests to convert the URLs to a unicode string again.